docs/MODULES.md

```diff
@@ -186,10 +211,12 @@ If you use this rule, there is no point in having any other Yggdrasil client rul
 
 You can specify a protocol, but that would limit the ports that are open.
 
-## Adding deprecated.systems peer info into CJDNS and Yggdrasil (Optional)
+## CJDNS and Yggdrasil public peering (Optional)
 
 #### CJDNS
 
+Other peers can be found in the [this](https://github.com/hyperboria/peers) repo of peers. Try to connect to only a few peers, and ones that are close to where you live. One peer is already shown below for you.
+
 Go to [Deprecated Systems](https://deprecated.systems/) website. You will see the following information:
```

Comment:
I think the docs should be more generic and not be linked to tomesh resources. 

Specifically listing deprecated.systems here may be inappropriate. I dont think we want these resources flooded with random people by advertising it so publicly.

Reply:
@benhylau  thoughts?

Reply:
It was like this before of course, but there's no harm in discussing it now. I tend to agree, it's easily abused by putting it here. I would just link to the public sites as I have done, and whether @benhylau wants to put the deprecated.systems stuff in the public repo is up to him.